### DISCUSSION TITLE
Introduce a bot to test downstream

### DISCUSSION THREAD

**Discussion by magic-akari** [swc-project/swc#8009] [Ideas]
It's better to have a bot like @typescript-bot with `test top100` supports.

**Comment by kdy1** [comment]
I have thought about this, but not sure about some steps.

 1. Search npm, finding packages with `@swc/core` as a dev dep.
 2. Ignore if the repository field in package.json does not exists
 3. Using the repository field, clone the repository.

But then how can we know the way to run tests?
